### ISSUE TITLE
New tutorials: write section to describe sources of packages

### ISSUE THREAD

**Issue by ddbeck** [pypa/python-packaging-user-guide#200]
(For more information about this issue, please see #194).

We need a short section to follow the "What's a package?" section (see #199) (i.e., as its own file). It should satisfy the following goals (in no particular order):

* explain that packages can be distributed through many channels (e.g., downloaded from PyPI with a tool like `pip`, an archive downloaded from a website, a VCS repository)
* explain that PyPI is an extremely popular way of getting packages but not the only way
* explain that PyPI is an index of packages, but not the only possible index
* illustrate the point with an example (maybe mention a popular package, then describe how it can be found on PyPI, in a VCS repository, and downloaded from the project's website)

In short, this section should answer the question, _Where do packages come from?_

When opening pull requests for this issue, please submit to the `develop` branch. If you have questions that concern contributing more generally, please use issue #194. Otherwise, feel free to comment with questions or feedback. Thanks!

**Comment by duims** [comment]
I'd be down to write this!

**Comment by ddbeck** [comment]
@duims Great, thanks! Feel to get started and please comment if you run into any roadblocks.
